recipes-ni/initscripts-nilrt/initscripts-nilrt_1.0.bb

```diff
@@ -78,6 +79,8 @@ do_install () {
 	install -m 0755 ${WORKDIR}/mountconfig           ${D}${sysconfdir}/init.d
 	# script for auto-loading kernel modules (needed for modules that are not auto-loaded by udev)
 	install -m 0755 ${WORKDIR}/modules_autoload      ${D}${sysconfdir}/init.d
+	# nicreatelogdir script that ensures log directory is present with proper ownership
+	install -m 0755 ${WORKDIR}/nicreatelogdir        ${D}${sysconfdir}/init.d
```

Comment:
This [recipe](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/recipes-support/syslog-ng/syslog-ng_3.%25.bbappend#L25) seems to be creating the same directory.